modes/basic-dev-mode/src/toolbarButtons.ts

```diff
@@ -9,74 +9,112 @@ function _createWwwcPreset(preset, title, subtitle) {
     id: preset.toString(),
     title,
     subtitle,
-    commands: [
-      {
-        commandName: 'setWindowLevel',
-        commandOptions: {
-          ...windowLevelPresets[preset],
-        },
-        context: 'CORNERSTONE',
+    commands: {
+      commandName: 'setWindowLevel',
+      commandOptions: {
+        ...windowLevelPresets[preset],
       },
-    ],
+      context: 'CORNERSTONE',
+    },
   };
 }
 
-function _createSetToolActiveCommands(toolName, toolGroupIds = ['default', 'mpr']) {
-  return toolGroupIds.map(toolGroupId => ({
-    commandName: 'setToolActive',
-    commandOptions: {
-      toolGroupId,
-      toolName,
-    },
-    context: 'CORNERSTONE',
-  }));
-}
+export const setToolActiveToolbar = {
+  commandName: 'setToolActive',
+  commandOptions: {
+    toolGroupIds: ['default', 'mpr'],
+  },
+  context: 'CORNERSTONE',
+};
 
 const toolbarButtons: Button[] = [
+  // sections
   {
-    id: 'MeasurementTools',
+    id: 'measurementSection',
```

Comment:
As suggested, this is still a button, so it uses the same casing as the other buttons.
The id of it is 'MeasurementTools' as a button.
The name of the button section to use is 'measurementSection' 
They are different objects and shouldn't use the same names.